fr/django_forms/README.md

```diff
@@ -281,7 +281,7 @@ Django va s'occuper de la validation : il va regarder si tous les champs de notr
 
 ## Éditer un formulaire
 
-Maintenant, nous savons comment ajouter un nouveau formulaire. Comment faire si nous voulons éditer un formulaire déjà existant ? C'est très similaire à ce que nous venons de faire. Tout d'abord, faisons rapidement quelques choses importantes. (Si vous ne comprenez pas quelque chose, vous devriez demander à votre entraîneur ou regarder les chapitres précédents, car nous avons déjà traité toutes ces étapes.)
+Maintenant, nous savons comme ajouter un nouveau post. Comment faire si nous voulons éditer un formulaire déjà existant ? C'est très similaire à ce que nous venons de faire. Tout d'abord, faisons rapidement quelques choses importantes. (Si vous ne comprenez pas quelque chose, vous devriez demander à votre entraîneur ou regarder les chapitres précédents, car nous avons déjà traité toutes ces étapes.)
```

Comment:
Hm, le changement `"comment" > "comme"` paraît incorrect, non ?
Quitte à changer "formulaire" pour "post", cela serait plus clair en le changeant également à la phrase suivante.

Reply:
Je plaide coupable pour «comme», c’est une erreur de frappe/d’inattention.

Pour le changement de formulaire pour post, je n’ai fait qu’être fidèle à l’original (qui utilise «post» pour cette phrase et «form» pour celles avant et après), je trouve ça plutôt étrange aussi mais je n’ai pas vraiment eu le courage de me replonger dans le contexte du tutoriel pour vérifier.

Cette phrase était marqué comme non-traduite dans CrowdIn donc j’imagine que le texte original utilisait «form» précédemment comme le reste du texte, du coup j’ai fait confiance au changement en anglais/à l’original.

Reply:
Hm, l'original dit actuellement "Now we know how to add a new post. But what if we want to edit an existing one?" du coup, il n'est pas ambigu. Je vote pour une traduction non ambigüe aussi, au choix:
- `Maintenant, nous savons comment ajouter un nouveau post. Comment faire si nous voulons éditer un post déjà existant ?`
- `Maintenant, nous savons comment ajouter un nouveau post. Comment faire pour en modifier un qui existe déjà ?`

Reply:
je pense que la 2e est bien, c’est assez sans répéter.

Reply:
Je peux faire le changement dans crowdin mais je connais pas le process pour faire une PR derrière. En tout cas, je sais pas si faire la modif côté github suffit ou si elle va se faire ecraser par crowdin (quelque chose me dit que c'est la 2e solution).

Reply:
<img width="541" alt="Capture d’écran 2020-05-09 à 20 57 12" src="https://user-images.githubusercontent.com/1457576/81482491-bc67d780-9237-11ea-9a37-14e220a23d05.png">
Voila